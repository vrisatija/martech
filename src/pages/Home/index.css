.content-container {
    backdrop-filter: blur(11px);
}
