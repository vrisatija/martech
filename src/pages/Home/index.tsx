import React from 'react';
import Images from '../../assets';
import CategoryTile from '../../components/CategoryTile';
import Header from '../../components/Header';
import { ROUTES } from '../../constants/routes';
import './index.css';

function Home(): JSX.Element {
  return (
    <div className="flex min-h-screen">
      <Header />
      <div
        className="flex justify-center flex-grow w-full bg-no-repeat bg-cover bg-center bg-opacity-80 mt-20"
        style={{
          backgroundImage: `url("${Images.backgroundImg}")`,
        }}
      >
        <div className="flex flex-col justify-center items-center content-container mx-20 my-10 px-10 py-18 bg-white-100 rounded-3xl 1.5xl:flex-row 1.5xl:m-auto laptop:px-24">
          <div className="text-white w-full mb-16 1.5xl:mr-16 1.5xl:mb-0">
            <div className="w-20 h-0.75 from-blue-100 bg-gradient-to-l to-blue-200 mb-4" />
            <div className="font-ibm-semi-bold text-5xl">
              MarTech
              <br />
              Compass
            </div>
            <div className="mt-5">Explore MarTechs for your clients</div>
          </div>
          <div className="flex justify-center flex-col md:flex-row">
            <CategoryTile
              categoryImg={Images.searchImg}
              title={(
                <>
                  SEARCH
                  <br />
                  FOR
                  <br />
                  VENDORS
                </>
              )}
              redirectionRoute={ROUTES.SEARCH}
            />
            <div className="flex flex-row items-center my-6 md:mx-4 md:my-0 md:flex-col">
              <div className="flex-1 border-t md:border-l" />
              <div className="mx-2 text-white">OR</div>
              <div className="flex-1 border-t md:border-l" />
            </div>
            <CategoryTile
              categoryImg={Images.compareImg}
              title={(
                <>
                  COMPARE
                  <br />
                  VENDORS
                </>
              )}
              redirectionRoute={ROUTES.COMPARE}
            />
          </div>
        </div>
      </div>
    </div>
  );
}

export default Home;
