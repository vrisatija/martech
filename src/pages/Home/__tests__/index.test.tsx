import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Home from '..';

describe('Home Screen', () => {
  it('renders home screen', () => {
    const HomeFragment = render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    ).asFragment();
    expect(HomeFragment).toMatchSnapshot();
  });
});
