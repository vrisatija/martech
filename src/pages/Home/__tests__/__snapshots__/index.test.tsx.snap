// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen renders home screen 1`] = `
<DocumentFragment>
  <div
    class="flex min-h-screen"
  >
    <div
      class="flex px-10 py-7 justify-between bg-blue-900 text-white absolute w-full"
    >
      <div
        class="font-ibm-semi-bold"
      >
        logo
      </div>
      <div
        class="flex items-center cursor-pointer"
      >
        <span
          class="mr-2"
        >
          Profile
        </span>
        <img
          alt="menu arrow"
          class="w-3.5"
          src="downArrow.svg"
        />
      </div>
    </div>
    <div
      class="flex justify-center flex-grow w-full bg-no-repeat bg-cover bg-center bg-opacity-80 mt-20"
      style="background-image: url(backgroundImg.png);"
    >
      <div
        class="flex flex-col justify-center items-center content-container mx-20 my-10 px-10 py-18 bg-white-100 rounded-3xl 1.5xl:flex-row 1.5xl:m-auto laptop:px-24"
      >
        <div
          class="text-white w-full mb-16 1.5xl:mr-16 1.5xl:mb-0"
        >
          <div
            class="w-20 h-0.75 from-blue-100 bg-gradient-to-l to-blue-200 mb-4"
          />
          <div
            class="font-ibm-semi-bold text-5xl"
          >
            MarTech
            <br />
            Compass
          </div>
          <div
            class="mt-5"
          >
            Explore MarTechs for your clients
          </div>
        </div>
        <div
          class="flex justify-center flex-col md:flex-row"
        >
          <button
            class="rounded-3xl w-76.25 h-91.75 transform transition duration-500 hover:scale-105"
            data-testid=""
            style="background-image: url(cardBackground.svg);"
            type="button"
          >
            <div
              class="px-10 pt-10"
            >
              <img
                alt="search artwork"
                src="searchImg.svg"
              />
              <div
                class="flex flex-col items-center mt-4 text-white font-poppins-semi-bold text-center"
              >
                SEARCH
                <br />
                FOR
                <br />
                VENDORS
              </div>
            </div>
          </button>
          <div
            class="flex flex-row items-center my-6 md:mx-4 md:my-0 md:flex-col"
          >
            <div
              class="flex-1 border-t md:border-l"
            />
            <div
              class="mx-2 text-white"
            >
              OR
            </div>
            <div
              class="flex-1 border-t md:border-l"
            />
          </div>
          <button
            class="rounded-3xl w-76.25 h-91.75 transform transition duration-500 hover:scale-105"
            data-testid=""
            style="background-image: url(cardBackground.svg);"
            type="button"
          >
            <div
              class="px-10 pt-10"
            >
              <img
                alt="search artwork"
                src="compareImg.svg"
              />
              <div
                class="flex flex-col items-center mt-4 text-white font-poppins-semi-bold text-center"
              >
                COMPARE
                <br />
                VENDORS
              </div>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
